doc/source/development.rst

```diff
@@ -174,19 +159,13 @@ If you want to see the available targets, use the option `-l` to let
 .. code:: shell-session
 
     $ tox -l
-    check
-    unit_py3_7
-    unit_py3_6
-    unit_py3_4
-    unit_py2_7
-    packagedoc
 
 To only run a special target, use the `-e` option. The following
 example runs the test cases for the Python 3.7 interpreter only:
```

Comment:
You mention "Python 3.7", but the tox call in the next line uses `unit_py3_6`. Seems like a contradiction, right? :wink:

Reply:
yes that was a mistake, great catch